types/evaluatex/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for evaluatex 2.2
+// Project: https://github.com/arthanzel/evaluatex
+// Definitions by: Fawaz Orabi <https://github.com/forabi-cosuno>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type Token =
+    | {
+          type: 'NUMBER' | 'POWER' | 'DIVIDE' | 'LPAREN' | 'RPAREN' | 'COMMAND';
+          value: string | number;
+          name: string | null;
+      }
+      | {
+        type: 'COMMAND';
+        value(params: unknown[]): unknown;
+        name: string | null;
+      }
+    | { type: 'SYMBOL'; value: string; name: null };
+
+interface EvaluatexResult {
+    /**
+     * @param variables a map of variables that can change between invocations of fn.
+     * @returns the numerical result of the calculation.
+     */
+    (variables?: Record<string, number>): number;
+    tokens: Token[];
+    expression: string;
+}
+
+declare function evaluatex(
+    /** an ASCII or LaTeX expression to be parsed and evaluated. */
+    expression: string,
+    /** a map of constant values - values that don't change if you invoke fn more than once. */
+    constants?: Record<string, number>,
+    /** a map of options for the compiler. */
+    options?: { latex?: boolean },
+): EvaluatexResult;
+
+export default evaluatex;
```

Comment:
```suggestion
export = evaluatex;
```
https://github.com/arthanzel/evaluatex/blob/master/index.js#L16
as per readme:
```ts
const evaluatex = require("evaluatex");
```
yes, it also support the other import. When consumed by TS users, this module would work with 'import ... from `...`' as per default settings in modern TS (esModuleInterop enabled by default)